metadata.yaml

```diff
@@ -1,50 +1,31 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
-# This file populates the Overview on Charmhub.
-# See https://juju.is/docs/sdk/metadata-reference for a checklist and guidance.
-
-# The charm package name, no spaces (required)
-# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
-name: is-charms-template
- 
-# The following metadata are human-readable and will be published prominently on Charmhub.
-
-# (Recommended)
-display-name: Charm Template
-
-# (Required)
-summary: A very short one-line summary of the charm.
+name: smtp-integrator
```

Comment:
Is the intention for this charm to be k8s only? In that case, we should add a suffix '-k8s' at the end of the name.

Reply:
No, it can be deployed in both substrates